zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -39,7 +39,8 @@ paths:
         - Job
       summary: Activate jobs
       description: |
-        Iterate through all known partitions and activate jobs up to the requested maximum.
+        Iterate through all known partitions and activate up to the requested maximum and
+        stream them back to the client as they are activated.
```

Comment:
❌ Looks like something went wrong with merging here. Please don't adjust this back to the old value.